starport/templates/typed/stargate/component/x/{{moduleName}}/keeper/{{typeName}}_test.go.plush

```diff
@@ -11,8 +11,8 @@ import (
 func createN<%= TypeName.UpperCamel %>(keeper *Keeper, ctx sdk.Context, n int) []types.<%= TypeName.UpperCamel %> {
 	items := make([]types.<%= TypeName.UpperCamel %>, n)
 	for i := range items {
-		items[i].<%= MsgSigner.UpperCamel %> = "any"
-		items[i].Id = keeper.Append<%= TypeName.UpperCamel%>(ctx, items[i])
+		<%= if (!NoMessage) { %> items[i].<%= MsgSigner.UpperCamel %> = "any"
```

Comment:
Same as a previous comment.

I think we can simply remove all "any" attribution